modules/apps/ccv/types/codec.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+)
+
+// RegisterLegacyAminoCodec registers the necessary x/ibc transfer interfaces and concrete types
+// on the provided LegacyAmino codec. These types are used for Amino JSON serialization.
+func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
```

Comment:
This function can be removed. Amino is only needed to enable ledger signed txs but I don't see any messages in this module